R/run.R

```diff
@@ -55,21 +56,31 @@ run <- function(input,
   approved_pkgs_url = options::opt("approved_pkgs_url", env = "whirl")
 
   # Message when initiating
-  d <- cli::cli_div(theme = list(rule = list(
-    color = "skyblue3", "line-type" = "double"
-  )))
-
-  zephyr::msg("Executing scripts and generating logs",
-              levels_to_write = c("verbose", "minimal"),
+  d <- NULL
+  zephyr::msg(message = "Executing scripts and generating logs",
+              theme = list(
+                rule = list(color = "skyblue3", "line-type" = "double")
+                ),
+              levels_to_write = c("verbose"),
               verbosity_level = verbosity_level,
-              msg_fun = cli::cli_rule)
+              msg_fun = \(message, theme, .envir) {
+                d <<- cli::cli_div(theme = theme, .auto_close = FALSE)
```

Comment:
It should. 
I am only using it here since I need to pass the div id on to the `on.exit` call later.
The `d` variable is initiated inside the function just before, so the `<<-` assignment should only go there and not outside of the function.
Reason for this change, and using `zephyr::msg` also for the div is that it would otherwise still be made no matter the verbosity_level.